subgraph/origin/schema.graphql

```diff
@@ -0,0 +1,9 @@
+type AvailableStateRoot @entity {
+  id: ID!
+  blockHeight: BigInt! # uint256
+  stateRoot: Bytes! # bytes32
+  blockNumber: BigInt! # uint256
```

Comment:
Can add documentation to differentiate between blockHeight and blockNumber.

Reply:
I have updated `blockHeight` to `anchoredBlockNumber` which is more verbose and clear.
Also we should not be using `height` as postfix since it's reserved for metablock.